src/eventTracking.js

```diff
@@ -94,6 +94,22 @@ export const SUBSCRIPTION_EVENTS = {
 
 export const LEARNER_CREDIT_MANAGEMENT_EVENTS = {
   TAB_CHANGED: `${LEARNER_CREDIT_MANAGEMENT_PREFIX}.budget-detail.tab.changed`,
+  EMPTY_STATE_CTA: `${LEARNER_CREDIT_MANAGEMENT_PREFIX}.budget-detail.empty-state-cta-to-catalog.clicked`,
+  VIEW_COURSE: `${LEARNER_CREDIT_MANAGEMENT_PREFIX}.catalog-search.view-course.clicked`,
+  // Assignment Modal
+  TOGGLE_NEXT_STEPS: `${LEARNER_CREDIT_MANAGEMENT_PREFIX}.assignment-modal.next-steps-collapsible.toggled`,
+  TOGGLE_IMPACT_ON_YOUR_LEARNERS: `${LEARNER_CREDIT_MANAGEMENT_PREFIX}.assignment-modal.impact-on-your-learners-collapsible.toggled`,
+  TOGGLE_MANAGING_THIS_ASSIGNMENT: `${LEARNER_CREDIT_MANAGEMENT_PREFIX}.assignment-modal.managing-this-assignment-collapsible.toggled`,
+  OPEN_ASSIGNMENT_MODAL_ASSIGN_COURSE: `${LEARNER_CREDIT_MANAGEMENT_PREFIX}.catalog-search.assign-course-cta.clicked`,
+  CLOSE_ASSIGNMENT_MODAL_ASSIGNED: `${LEARNER_CREDIT_MANAGEMENT_PREFIX}.assignment-modal.close-modal-assigned.clicked`,
+  CLOSE_ASSIGNMENT_MODAL_CANCEL: `${LEARNER_CREDIT_MANAGEMENT_PREFIX}.assignment-modal.close-modal-cancel.clicked`,
+  CLOSE_ASSIGNMENT_MODAL_EXIT: `${LEARNER_CREDIT_MANAGEMENT_PREFIX}.assignment-modal.close-modal-exit.clicked`,
+  CLOSE_ASSIGNMENT_MODAL_ASSIGNMENT_ALLOCATION_ERROR: `${LEARNER_CREDIT_MANAGEMENT_PREFIX}.assignment-modal.exit-assignment-allocation-modal.clicked`,
+  // Successful allocation
+  ASSIGNMENT_ALLOCATED_LEARNER_COUNT: `${LEARNER_CREDIT_MANAGEMENT_PREFIX}.assignment-modal.assignment-allocation.assigned`,
+  // Input Validation and API error allocation
+  EMAIL_ADDRESS_VALIDATION: `${LEARNER_CREDIT_MANAGEMENT_PREFIX}.assignment-modal.email-validation.changed`,
+  ASSIGNMENT_ALLOCATION_ERROR: `${LEARNER_CREDIT_MANAGEMENT_PREFIX}.assignment-modal.assignment-allocation.errored`,
 };
```

Comment:
nit: Something I just realized... these and many our existing event names have mixed usage of `_` and `-` to separate works. I wonder if we should ensure at least in these `LEARNER_CREDIT_MANAGEMENT_EVENTS` that we're only using either `_` OR `-`. That way, when we go to query against these events in a Snowflake dashboard, it won't be as confusing since the event name would contain only `_` or only `-`.

I.e., The `LEARNER_CREDIT_MANAGEMENT_PREFIX` relies on `_` but the concatenated event specific strings rely on `-`.

Reply:
opting for the `_` over `-` since according to the previous convention of content highlights, we are also using underscore, along with all the prefixes